doc/nrf/releases/release-notes-1.9.0.rst

```diff
@@ -0,0 +1,864 @@
+.. _ncs_release_notes_190:
+
+|NCS| v1.9.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF21, nRF52, nRF53, and nRF91 series.
+The SDK includes open source projects (TF-M, MCUboot OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and re-distributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+The feature is made available in its current state though the design and interfaces can change between release tags.
+The feature will also be labelled with "EXPERIMENTAL" in Kconfig files to indicate this status.
+Build warnings will be generated to indicate when features labelled EXPERIMENTAL are included in builds unless the Kconfig option :kconfig:`CONFIG_WARN_EXPERIMENTAL` is disabled.
+
+Highlights
+**********
+
+* Added experimental support for Nordic Distance Toolbox with examples. The software package adds phase-based distance measurement support for the nRF52 and nRF53 Series of devices.
+* Added support for Amazon Frustration-Free Setup (FFS) for Matter devices.
+* Added deep power-down mode to QSPI driver to optimize energy usage for solutions using an external flash memory.
+* Added support for micro:bit v2 board.
+* Added support for filtered ephemeris that improves A-GPS download size.
+* Updated the support for Periodic Advertising with the SoftDevice Controller. It is no longer experimental.
+* Updated the Zigbee solution on nRF5340. It is no longer experimental and is now Zigbee Platform Certification compliant.
+* Replaced proprietary sockets with nrf_modem APIs that include link-time optimization, thereby reducing memory usage.
+
+Sign up for the `nRF Connect SDK v1.9.0 webinar`_ to know more about the new features.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.9.0**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions`_.
+
```

Comment:
Due to the pending deprecation of SES, I think we need a notice in the NCS v1.9.  Suggest:

IDE support
*************
nRF Connect SDK version 1.9.0 is supported by both [nRF Connect extensions for Visual Studio Code](https://nrfconnect.github.io/vscode-nrf-connect/) and Segger Embedded Studio Nordic Edition.  In future releases, documentation on the setup and usage of Segger Embedded Studio Nordic Edition will no longer be available and all references to IDE support will instruct users with nRF Connect extensions for Visual Studio Code.  In future releases, Segger Embedded Studio will no longer be tested with the build system and it is not recommended for new projects.

nRF Connect extensions for Visual Studio Code will be under continued active development and supported and tested with future nRF Connect SDK releases. It is recommended for all new projects.

Reply:
@jdstapes, thanks for the suggestion! I have added this section.